File: src/main/java/com/fastcampus/innercircle/survey_api/domain/request/AnswerRequest.java
package com.fastcampus.innercircle.survey_api.domain.request;

public class AnswerRequest {

    private final Long questionId;  // 설문 받을 항목
    private final String content;   // 응답 값

    public Long getQuestionId() {
        return questionId;
    }

    public String getContent() {
        return content;
    }

    public AnswerRequest(Long questionId, String content) {
        this.questionId = questionId;
        this.content = content;
    }
}

File: src/main/kotlin/org/example/survey/dto/feedback/request/AnswerRequest.kt
package org.example.survey.dto.feedback.request

import org.example.survey.domain.Answer

data class AnswerRequest(
    val content: String,
    val questionId: Long,
) {
    fun toAnswerEntity(): Answer =
        Answer(
            content = this.content,
            questionId = this.questionId,
        )
}
